crates/iota-node/src/metrics.rs

```diff
@@ -1,15 +1,164 @@
 // Copyright (c) Mysten Labs, Inc.
 // Modifications Copyright (c) 2024 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
```

Comment:
Newline still missing here

Reply:
Fixed in [896b1a4](https://github.com/iotaledger/iota/pull/4391/commits/896b1a4724fbeec4b24d6c8003ab57496fb822b6).